### PR TITLE
Insurance nav items

### PR DESCRIPTION
**Pull request by darkodosenovic**
Adding Insurance navigation items

### CODE REVIEW

**Comment by rubene** [comment]
:book:

File: lib/ama_layout/navigation.yml [hunk 1]
```diff
@@ -18,6 +18,14 @@ member:
   - text: "Driver Education"
     link: "<%= Rails.configuration.driveredonline_site %>/login"
     icon: "fa-car"
+  - text: "Insurance"
```
Comment (rubene) [inline lib/ama_layout/navigation.yml]: will you able to get into insurance if the user is in renewal?

**Comment by darkodosenovic** [comment]
:wave:

**Comment by weyermannx** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**